### DISCUSSION TITLE
Layer on "a" instead of CapsLock

### DISCUSSION THREAD

**Discussion by guettli** [kmonad/kmonad#809] [Q&A]
First: Thank you for Kmonad. I used input-remapper in the past, but kmonad is more flexible.

I thought I am smart, and I use the CapsLock key for a new layer.

I am mostly interested in navigation like (pos1, end, del, up/down, pageUp/Down).

Now I realized that I need to move my pinkie finger to the side if I use CapsLock.

... why not create a new layer by pressing (and holding) "a"?

Then my pinkie finger does not need to move.

Of course, just pressing "a" should still be possible.

Which config is needed to create an "on hold layer"?

Example: holding A and pressing J should be like backspace. But of course, just pressing A should emit "a".

I don't need the default of keyboards: If I hold "a" then it gets emitted again and again. If I want to write "a" ten times, I can press "a" ten times. A new layer would give me much more features.

Same question on Reddit: https://www.reddit.com/r/Kmonad/comments/1aht1eq/layer_on_a_instead_of_capslock/

**Comment by jokesper** [comment]
take a look at tap-hold-next-release and it's variants in the tutorial.
